eng/pipelines/libraries/base-job.yml

```diff
@@ -96,7 +96,12 @@ jobs:
 
         - ${{ if ne(parameters.liveRuntimeBuildConfig, '') }}:
           - _runtimeDownloadPath: '$(Build.SourcesDirectory)/artifacts/transport/${{ parameters.runtimeFlavor }}'
-          - _runtimeArtifactName: '$(runtimeFlavorName)Product_${{ parameters.osGroup }}${{ parameters.osSubgroup }}_${{ parameters.archType }}_${{ parameters.liveRuntimeBuildConfig }}'
+          # Use corelib dependencies for coreclr
+          - ${{ if ne(parameters.runtimeFlavor, 'mono') }}:
+            - _runtimeArtifactName: 'CoreLib_${{ parameters.osGroup }}${{ parameters.osSubgroup }}_${{ parameters.archType }}_${{ parameters.liveRuntimeBuildConfig }}'
```

Comment:
I think this will break `libraries/run-test-job` for when runtimeFlavor 'coreclr'. I think we should add a variable that we set to: $(runtimeFlavorName)Product in `libraries/run-test-job` and then in `libraries/build-job` and `libraries/build-test-job` we set its value to `CoreLib_`.

Maybe the variable name can be `_coreclrArtifactPrefix`.

Reply:
now check for testScope to determine whether to use corelib or coreclr